src/website/src/pages/ec/components/button/docs/usage.mdx

```diff
@@ -3,39 +3,74 @@ title: Usage
-| Secondary button      | ![Button Image](https://inno-ecl.s3.amazonaws.com/media/images/EC/Buttons/Button_Secondary_Default.png) | or secondary actions.                                                                         |
-| Ghost button          | ![Button Image](https://inno-ecl.s3.amazonaws.com/media/images/EC/Buttons/Button_Ghost_Default.png)     | when an action does not require dominance on a page, for instance to "cancel" a form.         |
-| Search button         | ![Button Image](https://inno-ecl.s3.amazonaws.com/media/images/EC/Buttons/Button_Search_Default.png)    | use exclusively for the search function.                                                      |
+| Name  | Mandatory |
+| ----- | :-------: |
+| Label |     X     |
+| Icon  |           |
 
-Each page can have one to two Primary (or Call-to-action) buttons. Any remaining calls-to-action should be represented as Secondary or Ghost buttons.
+Buttons are defined by 3 major attributes:
 
 ### Button labels
 
-Button labels tell users what will happen when they click the button. Use verbs that describe the action, such as Send, Add or Delete. Do not use capitalization on buttons (only the first word in a phrase and proper nouns are capitalized) and no more than three words for button labels.
+Button labels tell users what will happen when they click the button.
+
+- Should be clear and distinct.
+- Use verbs that describe the action, such as Send, Add or Delete.
+- Keep it short, no more than 3 words.
 
 ### Button icons
 
-It is possible to add an icon to a button to enhance the action.
+Choose an icon that reinforce or clarify the meaning of a label in that particular context.
+Icons can be on right or left of your label.
+
+### Colour
+
+Ensure that the button background colour or label colour in the case of the ghost button provides sufficient contrast with the context/section background.
+
+## When to use
+
+To trigger an action. _ Actions which affect the website's state are buttons. _
+Actions include: Search, Submit... etc.
+
+An action is performed when the user interact actively with the website, ex: the user post a comment.
 
-| Button name           | Visual style                                                                                               |
-| --------------------- | ---------------------------------------------------------------------------------------------------------- |
-| Call-to-action button | ![Button Image](https://inno-ecl.s3.amazonaws.com/media/images/EC/Buttons/Button_CTA-Icon_Default.png)     |
-| Primary button        | ![Button Image](https://inno-ecl.s3.amazonaws.com/media/images/EC/Buttons/Button_Primary_icon_Default.png) |
+ECL has five buttons with different visual designs to make a distinction between the kind of action the button will carry out.
```

Comment:
```suggestion
There are five types of buttons with different visual clues.
```

Reply:
This changes the meaning. We have 5 buttons for 5 different cases.

Reply:
There are five button styles. (Bootstrap)